.cicd/run.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
```

Comment:
The fact that we have to hard-code the `.cicd/helpers` directory to load a script to programmatically define the location of the helpers directory reinforces my point that this is unnecessary complexity.

Reply:
I disagree. Once you load general, you should have consistent variables (ROOT_DIR, CICD_DIR, etc) that are then used throughout the scripts. If we change the directory structure or move scripts, it's WAY less chance that human error occurs by not updating hard coded paths.